tests/kas_messaging.spec.ts

```diff
@@ -201,3 +202,30 @@ for (const filter of filters) {
     }
   });
 }
+// test_6acl.py test_kafka_create_consumer_group_and_check_dashboard
+test('create consumer group and check dashboard', async ({ page }) => {
+  const instanceLinkSelector = page.getByText(testInstanceName);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await navigateToKafkaList(page);
+  await waitForKafkaReady(page, testInstanceName);
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Connection').click();
+
+  const bootstrapUrl = await getBootstrapUrl(page, testInstanceName);
+  console.log('bootstrapUrl: ' + bootstrapUrl);
+
+  // Consumer
+  await navigateToAccess(page, testInstanceName);
+  await grantConsumerAccess(page, credentials.clientID, testTopicName, consumerGroupId);
+  const consumer = new KafkaConsumer(bootstrapUrl, consumerGroupId, credentials.clientID, credentials.clientSecret);
+  const consumerResponse = await consumer.consumeMessages(testTopicName, expectedMessageCount);
```

Comment:
could you use retry mechanism for consumer?

Reply:
sure

Reply:
added @Frawless